releasenotes/notes/add-warning-when-hostname-not-used-as-canonical-b520c6f13758e58e.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add a warning when the hostname defined in the configuration will not be used as a canonical hostname.
```

`canonical` only has meaning for us I think, not for our end users:

```suggestion
    Log a warning when the hostname defined in the configuration will not be used as the in-app hostname.
```